bpf/bpf_netdev.c

```diff
@@ -527,12 +529,13 @@ static __always_inline int do_netdev_encrypt(struct __sk_buff *skb)
 		ret = DROP_WRITE_ERROR;
 		goto drop_err;
 	}
+	encrypt_iface = fib_params.ifindex;
```

Comment:
Does this need `ENCRYPT_NODE` ifdef protection?

Reply:
It will be set but unused if ENCRYPT_NODE is not defined.

Reply:
We could wrap it or not it shouldn't matter one way or the other I didn't ifdef the arg into the function to avoid having multiple ifdef/else/endif statements floating around.